3rdparty/expat.vcxproj

```diff
@@ -37,6 +37,7 @@
     <PlatformToolset Condition="'$(VisualStudioVersion)'=='15'">v141_xp</PlatformToolset>
     <PlatformToolset Condition="'$(VisualStudioVersion)'=='16'">v142</PlatformToolset>
     <XPDeprecationWarning>false</XPDeprecationWarning>
+    <PlatformToolset>v143</PlatformToolset>
```

Comment:
I'm pretty sure it has to be:
`<PlatformToolset Condition="'$(VisualStudioVersion)'=='17'">v143</PlatformToolset>`
here and everywhere below

Reply:
Updated that everywhere